javascript/linked.list/linked-list-implementation/README.md

```diff
@@ -0,0 +1,14 @@
+# Singly Linked List
+<!-- Short summary or background information -->
+This is working with linked lists and the methods we would use to add to them... and stringify them an to check if a linked lists includes a certain element
+
+## Challenge
+<!-- Description of the challenge -->
+This challenge was to create those methods and to test them
+
```

Comment:
I would out the name of each of the methods, and a quick description of what each functions does.